### ISSUE TITLE
Incorrectly labelled delimiter field in csv/xml export popup

### ISSUE THREAD

**Issue by sesamepaste** [sequelpro/sequelpro#3389]
<img width="227" alt="screenshot 2019-02-19 at 2 12 32 pm" src="https://user-images.githubusercontent.com/12541458/53001417-8aaafb00-3450-11e9-9dfe-8b4e246938f6.png">

The first field in csv options should be labeled delimiter instead of terminate.

**Comment by sesamepaste** [comment]
To reproduce run any query,  click on export csv option.

**Comment by dmoagx** [comment]
@Kaspik I don't see how this is a bug?
Also the labels in the list are verbs because it is intended to read "Fields: Commas terminate"

**Comment by Kaspik** [comment]
@dmoagx  Hmmm, I wasn’t thinking about this that way. I understand now, on the other hand I also understand users feel this is bug.

It’s up to you :)

**Comment by dmoagx** [comment]
> I also understand users feel this is bug.

Please explain. Why is "terminate" not a valid label?
If we were to change it wouldn't "Separator" be a much more logical choice than "delimiter" - after all it is called "Comma *Separated* Values".

Also a single complaint in 10+ years that this feature has been around doesn't really qualify as "users" ;)

**Comment by Kaspik** [comment]
@dmoagx  It is valid! But separator and delimiter too.
It’s just funny as WiKi says “CSV file is delimited file...” - 
https://en.m.wikipedia.org/wiki/Comma-separated_values

Revert it, close this. :)

**Comment by sesamepaste** [comment]
@dmoagx , I didn't notice the section headers ie. (Fields, Lines) until mentioned here. From a UI perspective, I think it would be better to visually represent a level of nesting. I am glad that you haven't received any issue about this, but that does not reduce the need for improvement. 

What I propose visually :

Fields
       <dropdown> Terminate
       ...
Lines
       <dropdown> Terminate

basically - visual indentation

At least this way the user very clearly sees the section nesting and the distinction between the two Terminate labels becomes clear.
